packages/framer-motion/src/animation/__tests__/GroupPlaybackControls.test.ts

```diff
@@ -182,4 +183,20 @@ describe("GroupPlaybackControls", () => {
         expect(a.complete).toBeCalled()
         expect(b.complete).toBeCalled()
     })
+
+    test("Gets and sets speed on all animations", async () => {
+        const a = createTestAnimationControls({
+            speed: 2,
+        })
+        const b = createTestAnimationControls({
+            speed: 2,
+        })
+
+        const controls = new GroupPlaybackControls([a, b])
+
+        controls.speed = -1
```

Comment:
This is only testing the setting, no the getting, I think that would be better in a separate tests.